plugin/kubernetes/handler.go

```diff
@@ -44,14 +44,14 @@ func (k Kubernetes) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.M
 		records, extra, err = plugin.SRV(ctx, &k, zone, state, plugin.Options{})
 	case dns.TypeSOA:
 		records, err = plugin.SOA(ctx, &k, zone, state, plugin.Options{})
+	case dns.TypeAXFR, dns.TypeIXFR:
```

Comment:
nit: I would put these first, or, at least, not in the middle as xfr is distinctly different from the other queries